eth/tracers/api_blocktrace.go

```diff
@@ -333,6 +357,11 @@ func (api *API) getTxResult(env *traceEnv, state *state.StateDB, index int, bloc
 			rcfg.ScalarSlot:    {},
 		})
 	for addr, keys := range proofStorages {
+
```

Comment:
```suggestion
```

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/372/commits/7b92a484fe79dcae22e697cdfdab742f480acefb